parallel-cost-model.rst

```diff
@@ -24,16 +24,16 @@ single flat kernel. We shall return to the concept of flattening in a
 later chapter.
 
 When we shall understand how efficient an algorithm is, we shall build
-our analysis around the two concepts of work and span. These concepts
-are defined inductively over the various Futhark language constructs
-and we may therefore argue about work and span in a compositional
-way. For instance, if we want to know about the work required to
-execute the ``multable`` function, we need to know about how to
-compute the work for a call to the ``map`` SOAC, how to compute the
-work for the ``iota`` operation, how to compute the work for the
+our analysis around the two concepts of `work and span`_. These
```

Comment:
Making this a link is the only real change-everything else is the spacing reflowing.